### PR TITLE
Week6 完成!

### PR DESCRIPTION
**Pull request by u88803494**

### CODE REVIEW

File: homeworks/week6/hw1/style.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    margin: 0px 10px;
+    display:inline-block;
+}
+
+.homePage__navigation-item:hover {
+    color:white;
+    cursor: pointer;/*滑鼠變換*/
+}
+
+.homePage__Introduction {
```
Comment (aszx87410) [inline homeworks/week6/hw1/style.css:32]: 怎麼這邊的 I 突然變大寫

**Comment by aszx87410** [comment]
https://lidemy.github.io/mentor-program-3rd-u88803494/homeworks/week6/hw1/
https://lidemy.github.io/mentor-program-3rd-u88803494/homeworks/week6/hw2/

螢幕快照 2019-05-27 下午10 28 29

File: homeworks/week6/hw1/style.css (hunk 1 above)
Comment (u88803494) [reply homeworks/week6/hw1/style.css:32]: 沒留意到再修正

**Comment by u88803494** [comment]
ok 再去修正

**Pull request merged**